### PR TITLE
When ++/-- used in thread, reply with points in that thread

### LINKED ISSUE #71: Display points count in threads when ++/-- is used in a thread
**Is your feature request related to a problem? Please describe.**
When replying to a thread with `++` or `--`, the responses from the pluspl.us bot is posted in the main channel and not in the thread.

**Describe the solution you'd like**
I would like for `++` and `--` in threads to have points updates displayed in the thread, not in the main channel.

### PR DESCRIPTION
**Pull request by LukeHowellDev**
When pluspl.us is used in threaded messages, the points will be shown in that thread.

I am new to Slack development and I'm not certain how to test this update. If I can get some guidance on testing, I will be happy to test it out.

Fixes #71

### CODE REVIEW

File: plusplus/operations/slack_handler.py [hunk 1]
```diff
@@ -9,13 +9,32 @@
...
 def process_incoming_message(event_data, req):
     # ignore retries
     if req.headers.get('X-Slack-Retry-Reason'):
         return "Status: OK"
 
     event = event_data['event']
     subtype = event.get('subtype', '')
+
+    # is the message from a thread
+    thread_ts = subtype === 'message_replied' ? event.get('event_ts') : ''
```
Comment (Conway) [inline plusplus/operations/slack_handler.py]: I don't think this is valid in Python (ternaries in Python are done like `variable = value_if_true if condition else value_if_false`)

File: plusplus/operations/slack_handler.py [hunk 2]
```diff
@@ -45,11 +64,7 @@ def process_incoming_message(event_data, req):
         if not thing:
             thing = Thing(item=found_user.lower(), points=0, user=True, team_id=team.id)
         message = update_points(thing, operation, is_self=user==found_user)
-        team.slack_client().api_call(
-            "chat.postMessage",
-            channel=channel,
-            text=message
-        )
+        post_message(message, channel, thread_ts)
```
Comment (Conway) [inline plusplus/operations/slack_handler.py:67]: Thanks for refactoring this into a method :)

File: plusplus/operations/slack_handler.py [hunk 3]
```diff
@@ -9,13 +9,32 @@
 user_exp = re.compile(r"<@([A-Za-z0-9]+)> *(\+\+|\-\-|==)")
 thing_exp = re.compile(r"#([A-Za-z0-9\.\-_@$!\*\(\)\,\?\/%\\\^&\[\]\{\"':; ]+)(\+\+|\-\-|==)")
 
+def post_message(message, channel, thread_ts = ''):
```
Comment (Conway) [inline plusplus/operations/slack_handler.py]: Nit: Please change the last argument to `thread_ts=''` to match the code style

File: plusplus/operations/slack_handler.py [hunk 4]
```diff
@@ -9,13 +9,32 @@
...
+    if parent_ts === '':
```
Comment (Conway) [inline plusplus/operations/slack_handler.py]: This should be a double equals not a triple equals

**Review by Conway — CHANGES_REQUESTED** [review]
Thanks for making this pull request! Just a few minor changes are needed to make this valid Python so I can test it.

**Comment by LukeHowellDev** [comment]
@Conway Thanks for the feedback on this. I have pushed the requested changes.

**Comment by Conway** [comment]
Thanks for making those changes @LukeHowellDev! I will test the changes this week

**Pull request merged**